Author: Daniel Baumann <mail@daniel-baumann.ch>
Description:
 Clarify manpage that fusermount is used for unmounting only these days
 (Closes: #651556).

diff -Naurp fuse.orig/doc/fusermount.1 fuse/doc/fusermount.1
--- fuse.orig/doc/fusermount.1
+++ fuse/doc/fusermount.1
@@ -1,7 +1,7 @@
 .TH FUSERMOUNT 1 2011\-10\-23 2.8.6 "Filesystem in Userspace (FUSE)"
 
 .SH NAME
-\fBfusermount\fR \- mount and unmount FUSE filesystems
+\fBfusermount\fR \- unmount FUSE filesystems
 
 .SH SYNOPSIS
 \fBfusermount\fR [\fIOPTIONS\fR] \fIMOUNTPOINT\fR
@@ -9,7 +9,7 @@
 .SH DESCRIPTION
 Filesystem in Userspace (FUSE) is a simple interface for userspace programs to export a virtual filesystem to the Linux kernel. It also aims to provide a secure method for non privileged users to create and mount their own filesystem implementations.
 .PP
-\fBfusermount\fR is a program to mount and unmount FUSE filesystems.
+\fBfusermount\fR is a program to unmount FUSE filesystems.
 
 .SH OPTIONS
 .IP "\-h" 4
